site/content/entry/compatability-list-wiki-created.md

```diff
@@ -0,0 +1,15 @@
++++
+date = "2017-06-03T11:30:00-04:00"
+title = "Compatibility List/Wiki Created!"
+author = "Saphiresurf"
+forum = 1603
++++
+
+[Check it Out Here] (https://citra-emu.org/game/)
+
+As game compatibility has broadened and the popularity of Citra has risen there's continued to be a bit of a void for information in terms of what you can actually play with Citra. When you launch a game it's been a bit of a blind test when you launch a game or even when progressing through them unless you do your own research.
+
+In order to fill this void we've decided to create a new game compatibility wiki! The 3DS' game library is slowly going to be populated into this list and each game will be assigned a rating of bronze, silver, gold, or platinum based on how well and how smoothly they run in Citra. We'll have a section on each games page titled "Notes" that will be a wiki editable by anyone that will be specifically for issues you may have had running the game, things you may have done to circumvent those issues, or generally anything having to do with running the game on official builds of Citra.
+
+
+Look forward to this compatiblity list expanding as we continue to grow!
```

Comment:
A little nitpicky, but the sentence feels a bit roundabout, maybe instead of "grow" would "mature" be better?

Also, passive voice! Or maybe instead of "we", why not "Citra"?